### DISCUSSION TITLE
Analog / single train

### DISCUSSION THREAD

**Discussion by conaito** [positron96/LocoNetControlStation#15] [Q&A]
Is it may possible to use (with other firmware) to run a single train (analog without decoder) with that command station?

**Comment by positron96** [comment]
The hardware allows it for sure. 
As a minimal implementation you can use a basic analogWrite example and tune it for required pins.

**Comment by conaito** [comment]
any sample you my know on the web?
Am 10. Dez. 2021, 15:23 +0100 schrieb Paul Melnikov ***@***.***>:

**Comment by positron96** [comment]
I'm sure there are hundreds of examples of `analogWrite` on the web (there are also some under "Analog" examples in Arduino IDE), but none of them will work right out of the box for your task without tweaking. To produce something useful you really need to understand what they do and what you need to produce to make your locomotive run. It's also essential to read the schematic to know what pins to use.
